lucet-runtime/tests/instruction_counting.rs

```diff
@@ -33,7 +32,40 @@ pub fn check_instruction_counts() {
             );
             ent
         })
-        .collect();
+        .collect()
+}
+
+#[test]
+pub fn check_icnt_off() {
```

Comment:
:tada: effective test reuse!

Reply:
My thanks to you for writing a easy-to-extend test!